### PR TITLE
Add config for several popular GitLab instances to docs

### PR DESCRIPTION
**Pull request by hickford**
Workaround for #677

### CODE REVIEW

**Review by dscho — APPROVED** [review]
That looks great!

File: docs/gitlab.md [hunk 1]
```diff
@@ -21,6 +21,30 @@ To use on another instance, eg. `https://gitlab.example.com` requires setup and
     git config --global --unset-all credential.https://gitlab.example.com.provider
 ```
 
+### Config for popular instances
+
+For convenience, here is config for several popular GitLab instances:
```
Comment (ldennington) [inline docs/gitlab.md:26]:
```suggestion
For convenience, here are the config commands for several popular GitLab instances:
```

**Review by ldennington — APPROVED** [review]
Thanks for elevating this for users! ⭐

File: docs/gitlab.md [hunk 2]
```diff
@@ -21,6 +21,30 @@ To use on another instance, eg. `https://gitlab.example.com` requires setup and
     git config --global --unset-all credential.https://gitlab.example.com.provider
 ```
 
+### Config for popular instances
+
+For convenience, here are the config commands for several popular GitLab instances:
```
Comment (ldennington) [inline docs/gitlab.md:26]: It could also be a good idea to clarify that these instances are community-maintained and are not managed by the GCM project.

**Review by ldennington — CHANGES_REQUESTED** [review]
Just came up with one more comment, thank you for bearing with me!

**Review by ldennington — APPROVED** [review]
Looks good, thanks @hickford!

**Comment by hickford** [comment]
Brilliant, thanks for merging.

**Pull request merged**